model/src/main/kotlin/ScanSummary.kt

```diff
@@ -149,10 +149,9 @@ data class ScanSummary(
 }
 
 /**
- * Set the `affectedPath` for scan timeout errors if otherwise it would have been unset. This way scan results which
- * have been created before `affectedPath` was introduced will still have that property set.
+ * Set the `affectedPath` for scan timeout errors if otherwise it had been unset. This way scan results which have been
```

Comment:
I find the sentence still clunky, what about: "Set the `affectedPath` for scan timeout errors if it is `null`."
And maybe also add "the" before "`affectedPath`".

Reply:
> And maybe also add "the" before "`affectedPath`".

I guess you mean "remove" instead of "add"?

Reply:
No, I meant to add it in the second sentence.